src/platform/utilities/api/index.js

```diff
@@ -1,8 +1,68 @@
 import * as Sentry from '@sentry/browser';
 
 import environment from '../environment';
+import ENVIRONMENTS from '../../../site/constants/environments';
 import localStorage from '../storage/localStorage';
 
+let sessionTimeoutMinutesLengthSSO = 0;
+const SSOeEndpoint = () => {
+  let environmentPrefix;
+  switch (ENVIRONMENTS.BUILDTYPE) {
+    case ENVIRONMENTS.VAGOVSTAGING:
+      environmentPrefix = 'sqa.';
+      break;
+    case ENVIRONMENTS.VAGOVDEV:
+      environmentPrefix = 'int.';
+      break;
+    default:
+      environmentPrefix = '';
+  }
+
+  return `https://${environmentPrefix}eauth.va.gov/keepalive`;
+};
+
+function checkAndUpdateSSOeSession() {
+  const sessionExpiration = localStorage.getItem('sessionExpirationSSO');
+
+  if (sessionExpiration) {
+    const minutesUntilExpiration =
+      (sessionExpiration.getTime() - Date.now()) / (1000 * 60);
```

Comment:
Date logic with magic numbers concerns me a bit.  I can see that it's converting milliseconds to minutes, but I'm wondering if it would be better if we had some common utilities or constants to use for things like this.

Reply:
it looks like there is a `timeFromNow` utility that could work to make that calculation. I was hesitant to pull something like `moment` in here, since I _think_ that could have bundle size implications for a relatively small slice of functionality. should that not be a concern here? 

at the very least adding in a comment on the exact logic at play would be sensible

Reply:
I agree, I'm not sure that adding `moment` is the best idea either.